### ISSUE TITLE
standalone-firefox-debug dies early

### ISSUE THREAD

**Issue by beatscode** [SeleniumHQ/docker-selenium#365]
I'm trying to follow the debugging directions but the standalone firefox-debug node dies on start up.
When I log the container I see the following 

![image](https://cloud.githubusercontent.com/assets/1777648/21854721/d0ba0412-d7e9-11e6-98f5-390bdc4b310d.png)

Not sure if `/opt/bin/entry_point.sh: line 5: /opt/bin/clear_x_locks.sh: No such file or directory` line is the issue here.

---- 
Here is shortened docker-composer.yml entries

````
services:        
  selenium:
    image: selenium/hub:2.53.0
    ports:
      - "4444:4444"       
    container_name: selenium    

  firefoxnode:
    image: selenium/standalone-firefox-debug:2.53.0
    environment:
      - HUB_PORT_4444_TCP_ADDR=selenium
    ports:
      - "63900:5900"
    links:
      - selenium:hub      

````

Is there something wrong with my configuration?

**Comment by ddavison** [comment]
perhaps related to #227 . check that issue out

**Comment by diemol** [comment]
This was fixed in #459, this issue can be closed.

**Comment by diemol** [comment]
@ddavison can you please close this issue?
